private-templates-service/README.md

```diff
@@ -1,62 +1,63 @@
 [![Deploy to Azure](https://azurecomcdn.azureedge.net/mediahandler/acomblog/media/Default/blog/deploybutton.png)](https://azuredeploy.net/)
 
-# Quickstart
-These steps will get our end-to-end app running locally. If any steps fail, try the longer steps below.
 
-Prerequisites:
--   Git
--   [Node v12](https://nodejs.org/en/download/)
+| Branch | Build Status |
+| ------ | ------------ |
+|  dev   | [![Build Status](https://dev.azure.com/microsoft/AdaptiveCards/_apis/build/status/ACMS/Deploy%20to%20Web%20App?branchName=dev)](https://dev.azure.com/microsoft/AdaptiveCards/_build/latest?definitionId=50969&branchName=dev) |
+|  stage | [![Build Status](https://dev.azure.com/microsoft/AdaptiveCards/_apis/build/status/ACMS/Deploy%20to%20Web%20App?branchName=stage)](https://dev.azure.com/microsoft/AdaptiveCards/_build/latest?definitionId=50969&branchName=stage) |
 
-**1. Clone the reponsitory.**
+# Quickstart
 
-**2. Switch to the desired branch. The latest build is on `dev`**
+## Deploy to Azure
 
-**3. `cd adaptivecards-templates/private-templates-service/server` and run `npm run init-app`. This installs and links dependencies.**
+Prerequisites: 
 
-**4. Open `server/app.ts` and replace mongoose code lines 39 - 64 with the following:**
-```
-import { InMemoryDBProvider } from '../../adaptivecards-templating-service/src/storageproviders/InMemoryDBProvider';
+- Azure account
+- Azure Active Directory (AAD) App Registration (instructions below)
+- [Azure resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal)
+- Azure App Service Plan
 
-app.use(function (req, res, next) {
-  res.header("Access-Control-Allow-Origin", "*");
-  res.header(
-    "Access-Control-Allow-Headers",
-    "Origin, Content-Type, Accept, Authorization, api_key"
-  );
-  res.header(
-    "Access-Control-Allow-Methods",
-    "GET, POST, OPTIONS, DELETE"
-  );
-  next();
-});
-app.options('*', function (req, res) { res.sendStatus(200); });
+1. Click the blue '**Deploy to Azure**'. Select the subscription and resource group under which you wish to deploy ACMS to. 
+2. Enter in the URL the portal will be hosted at into the '**Sites_adaptivecms_name**' field. Make sure this URL is added under the '**Redirect URIs**' section of your AAD App Registration. Detailed instructions and screenshots are listed below. 
+3. Enter in the Azure Active Directory App Registration application (client) id into the '**App_id**' field. 
+4. Enter the id of your App Service Plan into the '**Server_farm_id**' field. 
+5. If 'Yes' is selected for '**Telemetry_opt_in**', we will collect feedback from your instance of ACMS using App Insights. 

```

Comment:
I believe Jess is writing a privacy statement for Telemetry. I think we should add "See Telemetry Privacy Statement" in this step if the Privacy Statement is added somewhere else in the README.

Reply:
Yep that sounds good, I'll add it in the PR for the privacy statement!